### ISSUE TITLE
Error in "exo compute nlb list"

### ISSUE THREAD

**Issue by tanwald** [exoscale/cli#464]
I use `until exo compute nlb list ...` to wait for the load-balancer to become ready. Until it becomes ready it produces the following error:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0xfc315d]

goroutine 42 [running]:
github.com/exoscale/cli/cmd.(*nlbListCmd).cmdRun.func2({0x1284a60, 0x8})
	github.com/exoscale/cli/cmd/nlb_list.go:80 +0x1fd
github.com/exoscale/cli/cmd.forEachZone.func1()
	github.com/exoscale/cli/cmd/request.go:225 +0x25
github.com/hashicorp/go-multierror.(*Group).Go.func1()
	github.com/hashicorp/go-multierror@v1.1.0/group.go:23 +0x67
created by github.com/hashicorp/go-multierror.(*Group).Go
	github.com/hashicorp/go-multierror@v1.1.0/group.go:20 +0x8d
```
It happens when I create a new sks cluster and there is no load-balancer available for my account...

exo 1.58.0 274d052 (egoscale 0.89.0)
fedora: 5.18.13-200.fc36.x86_64

**Comment by 7fELF** [comment]
Hello @tanwald

This has been fixed in the latest release: [1.59.1](https://github.com/exoscale/cli/releases/tag/v1.59.1)

Thank you for reporting the issue.

**Comment by tanwald** [comment]
Thanks!
